lib/tax_cloud/client.rb

```diff
@@ -40,7 +40,16 @@ def auth_params
       }
     end
 
-    def safe(&_block)
+    def client_params
+      { wsdl: TaxCloud::WSDL_URL }.tap do |params|
+        if TaxCloud.configuration
```

Comment:
Followed this line: https://github.com/mezuka/tax_cloud/blob/master/lib/tax_cloud/client.rb#L36